examples/resources/awscc_cloudwatch_composite_alarm/with_two_sub_alarms.tf

```diff
@@ -0,0 +1,48 @@
+resource "awscc_cloudwatch_composite_alarm" "example" {
+  alarm_name        = "example-composite-alarm"
+  alarm_description = "Example of a composite alarm with various actions"
+
+  alarm_actions             = [aws_sns_topic.example_alarm_actions.arn]
+  ok_actions                = [aws_sns_topic.example_ok_actions.arn]
+  insufficient_data_actions = [aws_sns_topic.example_insufficient_data_actions.arn]
+
+  alarm_rule = "ALARM(${aws_cloudwatch_metric_alarm.cpu_gte_80.alarm_name}) OR ALARM(${aws_cloudwatch_metric_alarm.status_gte_1.alarm_name})"
+}
+
+resource "aws_sns_topic" "example_alarm_actions" {
+  name = "example-alarm-actions"
+}
+
+resource "aws_sns_topic" "example_ok_actions" {
+  name = "example-ok-actions"
+}
+
+resource "aws_sns_topic" "example_insufficient_data_actions" {
+  name = "example-insufficient-data-actions"
+}
```

Comment:
You may want to remove this block of code to keep the core resource simple.

Reply:
I kept this block for completeness so that users can copy paste these blocks to launch a composite alarm with actions. Without this block users would have to code SNS Topics themselves and realise that AWSCC does not support SNS topics today.

Reply:
When users look at the line 5- 7 for aws_sns_topic.*, they would have assume to use SNS Topic. In additional, you have also mentioned the use of AWS Provider in the md doco file. Therefore, I think line 12 to 22 can be removed to keep it clean and simple.

Reply:
IMO, its still fine to keep the `aws_sns_topic` to complete the example, since its a key component of the resource attributes.